org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/IXMLDocumentProvider.java

```diff
@@ -31,4 +34,12 @@
 	 *         null otherwise.
 	 */
 	DOMDocument getDocument(String uri);
+	
+	/**
+	 * All known documents XML server is working with at the moment
+	 * @return XML documents
+	 */
+	default Collection<DOMDocument> allDocuments() {

```

Comment:
To be consistent with `getDocument`, I will rename to `getAllDocuments()`

Reply:
Fixed